firmware/targets/f7/furi_hal/furi_hal_crypto.c

```diff
@@ -276,6 +289,35 @@ bool furi_hal_crypto_store_unload_key(uint8_t slot) {
     return (shci_state == SHCI_Success);
 }
 
+bool furi_hal_crypto_load_key(const uint8_t* key, const uint8_t* iv) {
+    furi_assert(furi_hal_crypto_mutex);
+    furi_check(furi_mutex_acquire(furi_hal_crypto_mutex, FuriWaitForever) == FuriStatusOk);
+
+    furi_hal_bus_enable(FuriHalBusAES1);
```

Comment:
I'd set AES_CR_EN and never ever touch it till key unloaded.

Reply:
Except mode change

Reply:
The original code also sets CR_EN once en/decryption starts. In GCM it also needs to be re-enabled when switching between certain phases.

Reply:
k, make sense.